### PR TITLE
Release notes 0.1-alpha

### PR DESCRIPTION
**Pull request by pmanrubia**
Release notes for v0.1-alpha

This is the structure of the document:

## Main points:

- Goals
- Metrics
- Information that belongs to a Department. Next Steps
- Core attributes from the Content Store. Next Steps
- Google Analytics (Unique Views). Next Steps

## Extra information

- Last updated time more than 6 years ago
- State of Content Items
- Data Migration
- Nested Resources
- API
- UI: Admin template
- Infrastructure
- Moving the application forward. Open discussions

### CODE REVIEW

**Comment by pmanrubia** [comment]
I have added ce36a1cd3d5d619871d23b1edae09d7236149f2b to provide additional information about why the app sits in the backend infrastructure.

File: doc/releases/notes-v0.1-alpha.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+2. Use [GDS adapters](https://github.com/alphagov/gds-api-adapters) to query SearchAPI and Content Store. This is the [Trello card][2].
+2. Update the current association between Organisation and Content Items from `one-to-many` to a `many-to-many`. This is the [Trello card to accomplish this work][1]. We didn't do it in this iteration because we are focused on validating it with only one Department.
+
+## Core attributes from the Content Store
+
+We are **not going to replicate the information stored in the Content Store**, but we need a few attributes so a user can have  additional context for a particular URL:
+
+Steps to retrieve additional attributes from a Content Item:
+
+1. We use the `base_path` from the Search API and query the Content Store to retrieve: `title`, `description`, `public_updated_at`, `content_id`.
```
Comment (theleebriggs) [inline doc/releases/notes-v0.1-alpha.md]: And `document_type`

File: doc/releases/notes-v0.1-alpha.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+## Google Analytics (Unique Views)
+
+* We are **not going to replicate Google Analytics for obvious reasons**, but we need to pull some GA metrics so we can describe better how the content is performing.
+
+How to retrieve metrics from GA:
+
+1. Use Google API for ruby.
+2. We have a limit of 2000 requests per day, but as we are using batch requests, we are not exceeding our quota when pulling data for Public Health of England (4000 content items)
+
```
Comment (theleebriggs) [inline doc/releases/notes-v0.1-alpha.md]: Public Health England rather than "of" England.

File: doc/releases/notes-v0.1-alpha.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+### Next step
+
+1. We still need to investigate what is the best size for our batch requests; both in Ruby side and in Google Analytics. There is a [Trello card][3] to address this task.
+2. We would also need to add GA configuration to our app so we can track the usage of the application itself. This is the [Trello card][4].
+
+# Content Performance Manager - Additional notes
+
+## Last updated time more than 6 years ago
+
+Some Content Items have a `last-public-update-time` older than 6 years. Although this might look like a data issue, it could also be the case that a Content Item with a creation date of 5 years ago in the Content Store, and a `last-public-updated-time` of 25 years: sometimes we display content that was published 20 years ago and has not been updated since.
```
Comment (theleebriggs) [inline doc/releases/notes-v0.1-alpha.md]: `public_updated_at` if we want to refer to the actual field name.

**Comment by pmanrubia** [comment]
Thank you for your review @theleebriggs.
I have addressed your comments. Please, approve and merge once you are happy.

**Pull request merged**